test_runner/regress/test_import.py

```diff
@@ -265,7 +265,7 @@ def _import(
     tenant = TenantId.generate()
```

Comment:
I wonder if we also need to generate new timeline ID here?
```
timeline = TimelineId.generate()
```

Reply:
Hmm, I agree it's a bit weird to reuse the same timeline ID, but it works. Per the TODO comment, the reason that this generates a different tenant ID is that it reuses the same pageserver.